pkg/didcomm/common/service/did_comm_msg_test.go

```diff
@@ -104,6 +100,66 @@ func TestDIDCommMsgMap_Type(t *testing.T) {
 	}
 }
 
+func TestDIDCommMsgMap_Clone(t *testing.T) {
+	tests := []struct {
+		name     string
```

Comment:
not really important but `name` is not used, unless you use it in a test run like: t.Run(`tc.name`, func(t *testng.T){... })

Reply:
I will fix it

Reply:
Done!